### DISCUSSION TITLE
LocalHost missing code

### DISCUSSION THREAD

**Discussion by hht28** [community/community#36857] [Other Feature Feedback, Questions, & Ideas]
Hello, I am having trouble with my localhost as after merging two branches into one, there is some code that doesn't appear on the server. However, the actual code is still present in the files. Is there any way to fix this?

**Comment by jfullstackdev** [comment]
you mean the local development server ? 

you might be pushing in a branch still, that happened to me to on GitHub but still the same concept

**Reply by hht28** [reply]
Could you explain this to me?

**Comment by ossanna16** [comment]
Thank you for asking a great question @hht28, I'm glad you found our community 🙂
